lib/transaction.rb

```diff
@@ -0,0 +1,12 @@
+class Transaction
```

Comment:
Is `Transaction` a clear name given then an instance of `Transaction` holds many records?